### PR TITLE
Localize Microsoft.TestPlatform.build.dll

### PR DESCRIPTION
**Pull request by Faizan2304**
Issue: https://github.com/Microsoft/vstest/issues/861

### CODE REVIEW

File: src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.Build.csproj [hunk 1]
```diff
@@ -10,11 +10,27 @@
     <WarningsAsErrors>true</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Resources\Resources.resx" />
```
Comment (codito) [inline src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.Build.csproj]: How are the resources packaged?

Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Build/Microsoft.TestPlatform.Build.csproj]: Thanks @codito for catching this. Fixed in next commit.

**Comment by Faizan2304** [comment]
@dotnet-bot test Ubuntu16.04 / Release Build please

**Comment by Faizan2304** [comment]
@dotnet-bot test Ubuntu16.04 / Debug Build please

**Comment by Faizan2304** [comment]
@dotnet-bot test Ubuntu16.04 / Release Build please
@dotnet-bot test Ubuntu16.04 / Debug Build please

**Review by codito — APPROVED** [review]

**Comment by Faizan2304** [comment]
@dotnet-bot test Ubuntu16.04 / Release Build please
@dotnet-bot test Ubuntu16.04 / Debug Build please

**Pull request merged**